framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -150,6 +150,9 @@
     // org.checkerframework.framework.source.SourceChecker.report
     "warns",
 
+    // Make checker ignore the expression in dead branch
+    "notCheckDeadCode",
```

Comment:
How about `ignoreDeadCode` to have more symmetry with other `ignoreXXX` options?

Reply:
By convention, for each option we list the main point where that option is used. Please add such a reference.

Reply:
> By convention, for each option we list the main point where that option is used. Please add such a reference.

Thanks, already addressed the comment in [1d1c348](https://github.com/eisop/checker-framework/pull/633/commits/1d1c348e3d69bddc0c98407c486c819a8990c69c)

Reply:
> How about `ignoreDeadCode` to have more symmetry with other `ignoreXXX` options?

addressed this comment in [58f4773](https://github.com/eisop/checker-framework/pull/633/commits/58f4773275dfa60da4457ce76be60d3cb19b24aa)